src/structures/buffers/publickeyrsa.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use crate::tss2_esys::TPM2B_PUBLIC_KEY_RSA;
+use crate::{Error, Result, WrapperErrorKind};
+use log::error;
+use std::convert::TryFrom;
+/// Struct holding a data value.
```

Comment:
Probably worth mentioning that this is intended to hold "the largest RSA public key supported by the TPM"